ucsb-cs156-f24/team02-f24-06
# MJ - Create Page for Recommendation Requests

Closes #31 

Adds Create page for Recommendation Requests
Adds Tests for Create page
Changed Recommendation Request form to have a dropdown instead of checkbox for "done". 

Passes Stryker 

https://67218c0899dcd8753c78278a-mxbzetgdft.chromatic.com/?path=/docs/pages-recommendationrequest-recommendationrequestcreatepage--docs

To Test: 
The page should look like this at first:
<img width="1053" alt="image" src="https://github.com/user-attachments/assets/4c67ecfd-91a4-4c87-aa0d-feedf3fc4bd7">

Don't input anything, just press create, it should look like this:
<img width="1027" alt="image" src="https://github.com/user-attachments/assets/fa3b910b-16d2-47d8-8e08-62c0d1c532a3">

After you fill out all fields and press create, there should be a pop up on the top right similar to the one below:
<img width="997" alt="image" src="https://github.com/user-attachments/assets/b9c04409-643f-40b1-b9d7-d1136864b192">
